controllers/internalrequest/adapter_test.go

```diff
@@ -211,6 +212,8 @@ var _ = Describe("PipelineRun", Ordered, func() {
 		})
 
 		It("should mark the InternalRequest as running", func() {
+			adapter.internalRequest.Status.StartTime = nil
```

Comment:
Why is this here? The test was working before.

Reply:
Because I added `MarkRunning()`  in the line 320. Otherwise it breaks because StartTime is not set (as we added the `go.metrics` to `MarkFailed()`and `MarkSucceeded()`.

Reply:
Yeah... about that `MarkRunning` call, it should be there. Not every test needs that function to get called, so it's better to call it for tests that need it.